server/src/test/java/org/elasticsearch/indices/cluster/ClusterStateChanges.java

```diff
@@ -351,4 +351,9 @@ private ClusterState executeClusterStateUpdateTask(ClusterState state, Runnable
         assertThat(result[0], notNullValue());
         return result[0];
     }
+
+    @SuppressWarnings("unchecked")
+    private static <T, R, E extends Exception> CheckedFunction<T, R, E> anyCheckedFunction() {
```

Comment:
I thought we had added this to some test utility somewhere.

Reply:
That was for `anyActionListener`, but I've now extracted the other similar methods to a test class.